tests/test_serializer_nested.py

```diff
@@ -202,3 +204,23 @@ def test_nested_serializer_with_list_multipart(self):
 
         assert serializer.is_valid()
         assert serializer.validated_data['nested']['example'] == set([1, 2])
+
+
+class TestNestedWrites:
+    def test_nested_source_attr_write(self):
```

Comment:
Minor nitpick - I would change the test name to `test_dotted_source_write_failure`. 
- `nested` usually refers to nested serializers. Mixing the terminology is a little confusing.
- The existing test name sounds like dotted-source writes should succeed.